clients/drdisas/drdisas.cpp

```diff
@@ -169,7 +199,7 @@ main(int argc, const char *argv[])
     while (pc <= stop_pc) {
         // Check ahead of time to see whether this instruction enters the redzone
         // (or, we could disassemble into a buffer and check before printing it).
-        if (pc + decode_sizeof(dcontext, pc, NULL _IF_X86_64(NULL)) > stop_pc) {
+        if (pc + decode_sizeof(dcontext, pc, NULL _IF_X86_64(NULL)) > stop_pc + 1) {
```

Comment:
@derekbruening I removed the option away from arm for now in order to avoid keeping this PR pending. I also changed PR message to _issue_ instead of _fixes_.

Reply:
Okay with me merging? I'll wait for confirmation just in case you want to take another look.